### DISCUSSION TITLE
how to disable index download in arduino ide

### DISCUSSION THREAD

**Discussion by polys96** [earlephilhower/arduino-pico#2498] [Q&A] [answered]
hello, I am currently having an issue using arduino ide with "https://github.com/earlephilhower/arduino-pico/releases/download/global/package_rp2040_index.json" in additional boards is there any way to use the pico-core without waiting to download index?
![image](https://github.com/user-attachments/assets/0c65128f-e3c4-4e14-b273-1fa6d39018aa)

**Comment by earlephilhower** [comment, marked as answer]
You can remove the url from the boards manager and use git manually to install.   The Readme and docs have the commands.

**Reply by polys96** [reply]
thank you!
